app/components/Dashboard/PortfolioPanel/PortfolioBreakdownChart.jsx

```diff
@@ -63,7 +63,6 @@ export default class PortfolioBreakdownChart extends React.Component<Props> {
               id="totalWalletValue"
             />
           </Pie>
-          <Tooltip formatter={this.formatValue} />
```

Comment:
Struggled with trying to style and format this value but ultimately was unsuccessful will revisit at a later date if its deemed important